app/utils/nav-builder.js

```diff
@@ -94,13 +94,17 @@ const adminNavigationItems = (currentPath, permissions, type, paymentProvider, a
     {
       id: 'navigation-menu-payment-types',
       name: 'Card types',
-      url: formatAccountPathsFor(paths.paymentTypes.index, account.external_id),
-      current: pathLookup(currentPath, paths.paymentTypes.index),
+      url: formatAccountPathsFor(paths.account.paymentTypes.index, account.external_id),
+      current: pathLookup(currentPath, paths.account.paymentTypes.index),
       permissions: permissions.payment_types_read && type === 'card'
     }
   ]
 }
 
+// const pathMatches(url, paths) {
```

Comment:
Can be removed I think

Reply:
oops yeh thanks accidental WIP

Reply:
will remove in next PR just to save build time